openslides/utils/autoupdate.py

```diff
@@ -431,13 +87,16 @@ def inform_data_collection_element_list(collection_elements: List[CollectionElem
     Informs the autoupdate system about some collection elements. This is
     used just to send some data to all users.
     """
-    # Put all stuff into the autoupdate_bundle.
```

Comment:
Here the same as in inform_changed_data().

Reply:
done